packages/hooks/src/createHooks.js

```diff
@@ -12,8 +12,10 @@ import createDidHook from './createDidHook';
  * @return {Object} Object that contains all hooks.
  */
 function createHooks() {
-	const actions = {};
-	const filters = {};
+	const actions = Object.create( null );
```

Comment:
Not clear why this is changing this here, is this performance related or required for other changes here?

Reply:
Yes, from original comment:

>- Performant alternatives (truthy property access on null-prototype object vs. hasOwnProperty, faa0ed3, [jsperf](https://jsperf.com/object-create-null-vs-hasownproperty))
>   - The reason for Object.create( null ) is to avoid issues with prototype property access:
>      - `!! ( {} ).valueOf // true`
>      - `!! Object.create( null ).valueOf // false`

Reply:
Ok, got it - I didn't see that this was required for the other changes to work. Nice!